src/edtr-io/plugins/box/renderer.tsx

```diff
@@ -14,7 +14,7 @@ import { useInstanceData } from '@/contexts/instance-context'
 import { useLoggedInData } from '@/contexts/logged-in-data-context'
 import { hasOwnPropertyTs } from '@/helper/has-own-property-ts'
 
-const boxTypeStyle = {
+export const boxTypeStyle = {
   blank: {},
   example: {},
   quote: { icon: faQuoteRight },
```

Comment:
maybe moving this out of this file into the frontend will do the trick (and import from there)

Reply:
ah damn, was afraid that might happen. we really have to take care when we try to split editor plugins into a render and editor part.